server/channels/app/support_packet.go

```diff
@@ -78,21 +80,36 @@ func (a *App) generateSupportPacketYaml() (*model.FileData, string) {
 
 	uniqueUserCount, err := a.Srv().Store().User().Count(model.UserCountOptions{})
 	if err != nil {
-		return nil, errors.Wrap(err, "error while getting user count").Error()
+		rErr = multierror.Append(errors.Wrap(err, "error while getting user count"))
 	}
-
-	analytics, err := a.GetAnalytics("standard", "")
-	if analytics == nil {
-		return nil, errors.Wrap(err, "error while getting analytics").Error()
+	elasticPostIndexing, err := a.Srv().Store().Job().GetAllByTypePage(model.JobTypeElasticsearchPostIndexing, 0, 2)
+	if err != nil {
+		rErr = multierror.Append(errors.Wrap(err, "error while getting ES post indexing jobs"))
+	}
+	elasticPostAggregation, _ := a.Srv().Store().Job().GetAllByTypePage(model.JobTypeElasticsearchPostAggregation, 0, 2)
+	if err != nil {
+		rErr = multierror.Append(errors.Wrap(err, "error while getting ES post aggregation jobs"))
+	}
+	ldapSyncJobs, err := a.Srv().Store().Job().GetAllByTypePage(model.JobTypeLdapSync, 0, 2)
+	if err != nil {
+		rErr = multierror.Append(errors.Wrap(err, "error while getting LDAP sync jobs"))
+	}
+	messageExport, err := a.Srv().Store().Job().GetAllByTypePage(model.JobTypeMessageExport, 0, 2)
+	if err != nil {
+		rErr = multierror.Append(errors.Wrap(err, "error while getting message export jobs"))
+	}
+	dataRetentionJobs, err := a.Srv().Store().Job().GetAllByTypePage(model.JobTypeDataRetention, 0, 2)
+	if err != nil {
+		rErr = multierror.Append(errors.Wrap(err, "error while getting data retention jobs"))
+	}
+	complianceJobs, err := a.Srv().Store().Job().GetAllByTypePage("compliance", 0, 2)
+	if err != nil {
+		rErr = multierror.Append(errors.Wrap(err, "error while getting compliance jobs"))
+	}
```

Comment:
There is no `compliance` job in https://github.com/mattermost/mattermost/blob/e7c7ce2286a393734ec42ec7c208e6773d61d41d/server/public/model/job.go#L11-L38.

@coltoneshaw Which job type do you mean here? `message_export`?

Reply:
Hmm. I don't recall, very well could have been an error. Message export exists already, see below. I was trying to get a different job type that I pulled from the jobs table of a customer. I'm vauge and if it doesn't exist and was incorrect, just delete it. 

```
message_export_jobs:
- id: uqfwwzcqatng5gffmse6rmhmyh
  type: message_export
  priority: 0
  createat: 1693487253804
  startat: 0
  lastactivityat: 0
  status: pending
  progress: 0
  data: {}
  ```

Reply:
Deleted in f100553d94